Source/WebCore/platform/graphics/cocoa/SystemFontDatabaseCoreText.cpp

```diff
@@ -235,7 +235,7 @@ SystemFontDatabaseCoreText::CascadeListParameters SystemFontDatabaseCoreText::sy
     CascadeListParameters result;
     result.locale = description.computedLocale();
     result.size = description.computedSize();
-    result.italic = isItalic(description.italic());
+    result.italic = isItalic(description.fontStyleSlope());
```

Comment:
Seems unclear when to call `isItalic` and when to just check against `nullopt`.

Reply:
Agreed.